### DISCUSSION TITLE
Time-to-live in Transaction

### DISCUSSION THREAD

**Discussion by bigbenbeer** [MeshJS/mesh#22] [Q&A] [answered]
Simple question: Can I specify the time-to-live part of the transaction from Mesh? 

Within the [IOG cardano-wallet](https://input-output-hk.github.io/cardano-wallet/api/v2021-09-29/#operation/postTransaction) it is possible to specify a TTL value to prevent an infinite expiry transaction. How can I do that with Mesh?

**Comment by jinglescode** [comment, marked as answer]
Hey @bigbenbeer, yes indeed we can. We are excited to push out so many new features in 1.2, its currently available if you download the latest beta version. To specify the TTL, we added `resolveSlotNo` and `setTimeToExpire`. See `Set start and expire time` in [Transaction](https://mesh.martify.io/apis/transaction) on how to do it.

**Reply by bigbenbeer** [reply]
Thank you!
